src/services/identity/DeploymentsService.ts

```diff
@@ -115,6 +118,83 @@ class DeploymentsService {
           .map(() => amplifyInfo)
       })
   }
+
+  getDeploymentInfoFromSiteId = async (repoId: string) => {
```

Comment:
could we rename `repoId` to `siteId`? we have a `repos` and a `sites` table, might be better to make a distinction here